packages/components/src/Icon/Icon.mdx

```diff
@@ -305,6 +305,7 @@ import { Icon } from "@jobber/components/Icon";
 | <Icon name="presentation" /> | `presentation` |
 | <Icon name="quickbooks" />   | `quickbooks`   |
 | <Icon name="reports" />      | `reports`      |
+| <Icon name="signature" />    | `signature`    |
```

Comment:
Considered adding to the "system" category of Icons but I think "other" made more sense in the end

Reply:
Other is where I was expecting it to land too.